primitives/linux-port/primitiveprocessor.h

```diff
@@ -469,21 +471,20 @@ class PrimitiveProcessor
 // Bit pattern representing NULL value for given column type/width
 // TBD Use TypeHandler
 template <typename T, typename std::enable_if<sizeof(T) == sizeof(int128_t), T>::type* = nullptr>
-T getNullValue(uint8_t type)
+T getNullValue(uint8_t type, uint32 width)
 {
   return datatypes::Decimal128Null;
 }
 
 template <typename T, typename std::enable_if<sizeof(T) == sizeof(int64_t), T>::type* = nullptr>
-T getNullValue(uint8_t type)
+T getNullValue(uint8_t type, uint32 width)
```

Comment:
JFYI @mariadb-AlexanderPresniakov here the template uses ugly SFINAE(enable_if) to narrow the number of candidates found by the compiler. There were no concepts available when I refactored this part. Put it plain this template is triggered for types that are 8 bytes long. I might be missing some details but isn't this info enough to assume that the width will be 8 bytes?

Reply:
Thanks, I removed the width argument